day3/slice_timing.ipynb

```diff
@@ -1,397 +1,425 @@
 {
+ "cells": [
```

Comment:
This is one of the problems with notebooks - it's hard to see where the changes are.  I think what happened here is that you loaded this in a newer version of IPython and then saved it, so there aren't many (any?) real changes, just format changes.